neo/test/actor/NeoActorTest.scala

```diff
@@ -99,5 +115,53 @@ class NeoActorTest extends Specification {
         """.stripMargin)
       ok
     }
+
+    "handle HTTP request/response to Berry" in {
+      val url = "localhost:9000"
+      val interval = Interval.parse("2015-11-22T00:00:00.000-08:00/2017-02-14T17:44:56.000-08:00")
+      val jsonRequest = Json.parse(
+        s"""
+           |{
+           |  "dataset"     : "twitter.ds_tweet",
+           |  "keywords"    : ["zika"],
+           |  "timeInterval": ${JsObject(Seq(("start", JsNumber(interval.getStartMillis)), ("end", JsNumber(interval.getEndMillis))))},
+           |  "timeBin"     : "day",
+           |  "geoLevel"    : "state",
+           |  "geoIds"      : [37, 51],
+           |  "mergeResult" : ${JsBoolean(false)}
+           |}
+       """.stripMargin
+      )
+      val json1 = Json.obj("city" -> "NY", "count" -> 100)
+      val json2 = Json.obj("city" -> "LA", "count" -> 150)
+
+      val ws = mock[WSClient]
+      val wsRequest = mock[WSRequest]
+      val header = DefaultWSResponseHeaders(200, Map.empty)
+      val source: Source[ByteString, _] =
+        Source(List(ByteString(json1.toString()), ByteString("\n"), ByteString(json2.toString())))
+
+      when(ws.url(any)).thenReturn(wsRequest)
+      when(wsRequest.withMethod("POST")).thenReturn(wsRequest)
+      when(wsRequest.withBody[JsValue](any[JsValue])(any[Writeable[JsValue]])).thenReturn(wsRequest)
+      when(wsRequest.stream()).thenReturn(Future(StreamedResponse(header, source)))
+
+      val frontEnd = new TestProbe(system)
+      val neo = system.actorOf(NeoActor.props(frontEnd.ref, ws, url, Config.Default))
+      frontEnd.send(neo, jsonRequest)
+
+      val jsonExpected1a = Json.obj("key" -> Sample, "value" -> json1)
```

Comment:
can you rename these two variables? like `jsonExpected1a` --> `ExpectedSampleJson1` ?